homework-3/src/components/Forms/FormSignIn.js

```diff
@@ -19,35 +22,25 @@ const FormSignUp = () => {
         password: '',
       }}
       validate={(values) => {
-        const errors = {};
-        const regulPassword = /(?=.*[A-Z]+)(?=.*[0-9])(?=.*[a-z]){6,}/;
-        const regulEmail =
-          /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
-
         if (!regulPassword.test(values.password)) {
-          setPassword((isPassword = true));
+          setPassword(!isPassword);
         } else {
-          setPassword((isPassword = false));
+          setPassword(!isPassword);
```

Comment:
Logic does the same, recheck it will not work in a propper way

Reply:
Fixed